manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.yaml

```
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
+PackageVersion: v2.8
```

```suggestion
PackageVersion: 1.0.0
```
Must match control panel